Tribler/Test/test_multichain_community.py

```diff
@@ -0,0 +1,544 @@
+"""
+This file contains the tests for the community.py for MultiChain community.
+"""
+import time
+import uuid
+import logging
+
+from Tribler.Core.Session import Session
+
+from Tribler.community.multichain.community import  MultiChainCommunity, CRAWL_REQUEST, CRAWL_RESPONSE, CRAWL_RESUME
+from Tribler.community.multichain.conversion import EMPTY_HASH
+
+from Tribler.community.tunnel.routing import Circuit, RelayRoute
+from Tribler.community.tunnel.tunnel_community import TunnelExitSocket
+
+from Tribler.Test.test_as_server import BaseTestCase
+
+from Tribler.dispersy.tests.dispersytestclass import DispersyTestFunc
+from Tribler.dispersy.util import blocking_call_on_reactor_thread
+from Tribler.dispersy.tests.debugcommunity.node import DebugNode
+
+
+class TestMultiChainCommunity(DispersyTestFunc):
```

Comment:
I would rather make a subclass of `AbstractServer` since this test is part of the Tribler test suite. `AbstractServer` contains the watchdog and additional checks to verify whether the tests leave a clean reactor/leave no open sockets.